students/urtnowski_daniel/lesson_06_dicts_tuples_sets_args_kwargs/snakify_lesson_10.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python3
+
+"""
+snakify_lesson_10.py: Solutions for 3 of problems defined in:
+Lesson 10. Sets
+(https://snakify.org/lessons/sets/)
+"""
+
+__author__ = "Daniel Urtnowski"
+__version__ = "0.1"
+
+
+def read_set_of_integers(items_count):
+    new_set = set()
+
+    for i in range(items_count):
+        new_set.add(int(input()))
+
+    return new_set
+
+
+def print_set(input_set):
+    ordered = list(input_set)
+
+    ordered.sort()
```

Comment:
to jest dobry przypadek gdzie można użyć `sorted` bezpośrednio na `input_set`: https://stackoverflow.com/questions/22442378/what-is-the-difference-between-sortedlist-vs-list-sort